src/supervisor/gen_rpc_client_sup.erl

```diff
@@ -35,11 +35,11 @@ start_link() ->
 
 -spec start_child(node_or_tuple()) -> supervisor:startchild_ret().
 start_child(NodeOrTuple) when ?is_node_or_tuple(NodeOrTuple) ->
-    ?tp(debug, gen_rpc_starting_new_client, #{target => NodeOrTuple}),
+    ?tp(debug, gen_rpc_starting_new_client, #{target => NodeOrTuple, domain => ?D_CLIENT}),
     case supervisor:start_child(?MODULE, [NodeOrTuple]) of
         {error, {already_started, CPid}} ->
             %% If we've already started the child, terminate it and start anew
-            ?tp(warning, gen_rpc_restarting_client, #{target => NodeOrTuple, old_client => CPid}),
+            ?tp(warning, gen_rpc_restarting_client, #{target => NodeOrTuple, old_client => CPid, domain => ?D_CLIENT}),
             ok = stop_child(CPid),
             supervisor:start_child(?MODULE, [NodeOrTuple]);
```

Comment:
Nit, related to the original PR theme. I wonder if we need this piece of logic and if it makes things actually worse.